wallet/src/de/schildbach/wallet/data/AppDatabaseMigrations.kt

```diff
@@ -46,5 +47,15 @@ class AppDatabaseMigrations {
                         "USING FTS4(`name`, `manufacturer`, content=`atm`)")
             }
         }
+
+        @JvmStatic
+        val migration4To9 = object : Migration(4, 9) {
```

Comment:
Shouldn't this be 3 to 9? Also, it's possible that we don't need migration2to3 anymore.

Reply:
Hmm, good point I guess both assumptions are correct.
I'll make sure and correct.

Reply:
fixed